frontends/web/src/routes/new-settings/components/dropdowns/activecurrenciesdropdown.module.css

```diff
@@ -0,0 +1,58 @@
+.select :global(.react-select__input-container) {
+    position: absolute;
+    top: -2px;
+    left: 6px;
+}
+
+.select :global(.react-select__option) {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+}
+
+/*displays only first 3 currencies, the rest gets hidden*/
+.select :global(.react-select__multi-value):nth-child(n + 4) {
+    display: none;
+}
+
+
+/*
+displays ', ' as a pseudo component for all currency component
+except the last displayed component (the 3rd currency component).
+
+The reason for :nth-last-child(2) instead of just :last-child is because there's an extra
+component created by react-select after the actual last currency component. 
+
+So, we're targetting the second to last component here, 
+which is the last selected currency in the DOM.
+*/
+.select:not(.hideMultiSelect) :global(.react-select__multi-value):not(:nth-last-child(2))::after {
+    content: ', ';
+}
+
+/*
+displays '...' as a pseudo component of the 3rd currency and only when there's more than 3 selected.
+:nth-last-child(2) is used for the same reason as above
+*/
+.select:not(.hideMultiSelect) :global(.react-select__multi-value):nth-child(3):not(:nth-last-child(2))::after {
+    content: ' ...';
```

Comment:
This CSS selector is a bit crazy :) but I guess adding `…` via JSX would make the component a lot more complicated. then fine I guess…

remove space as it doesn't support leading space in content property and I think space is not needed from layout perspective.

and could you use `&hellip;` tripple dot `…` 🤓  ?